trunk-recorder/main.cc

```diff
@@ -893,6 +897,14 @@ void unit_ack(string unit_script, string shortName, long unit) {
   }
 }
 
+void unit_locreg(string unit_script, string shortName, long unit, long talkgroup) {
```

Comment:
What does this function do? It looks like it mostly calls an external script

Reply:
Yes, it calls the unitScript I added in #181/#441 on Location Registration Response messages (I noticed that I was getting them, and saw in the standards that they contained radio and talkgroup IDs).

Reply:
Gotcha - Let's leave this in? Can you also add short comment in on what the type of message is and what the function does?